taichi/ir/transforms.h

```diff
@@ -75,25 +75,29 @@ bool constant_fold(IRNode *root,
                    const CompileConfig &config,
                    const ConstantFoldPass::Args &args);
 void offload(IRNode *root, const CompileConfig &config);
+bool transform_statements(
+    IRNode *root,
+    std::function<bool(Stmt *)> filter,
+    std::function<void(Stmt *, DelayedIRModifier *)> transformer);
 /**
  * @param root The IR root to be traversed.
  * @param filter A function which tells if a statement need to be replaced.
  * @param generator If a statement s need to be replaced, generate a new
  * statement s1 with the argument s, insert s1 to s's place, and replace all
```

Comment:
nit: by saying "insert s1 to s's place", do we mean "insert |s1| to where |s| is defined, removes |s|'s definition and replaces all usages of ..."?

Reply:
Right, I'll rephrase this.